tagblock_test.go

```diff
@@ -0,0 +1,139 @@
+package nmea
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var tagblocktests = []struct {
+	name string
+	raw  string
+	err  string
+	msg  TagBlock
+}{
+	{
+
+		name: "Test NMEA tag block",
+		raw:  "\\s:Satelite_1,c:1553390539*62\\!AIVDM,1,1,,A,13M@ah0025QdPDTCOl`K6`nV00Sv,0*52",
+		msg: TagBlock{
+			Time:   1553390539,
+			Source: "Satelite_1",
+		},
+	},
+	{
+
+		name: "Test NMEA tag block with head",
+		raw:  "UdPbC?\\s:satelite,c:1564827317*25\\!AIVDM,1,1,,A,19NSRaP02A0fo91kwnaMKbjR08:J,0*15",
```

Comment:
from what I'm reading the `UdPbC` isn't actually part of the tag block. It's a prefix signalling that udp multicast is being used.

Reply:
Also, these, and the examples from NemaStudio are the first time I've seen that prefix. I've looked for any references in the 2nd hand accounts of what the spec actually says and nothing suggests that "head" is standard.

OTOH, I would hesitate to remove this without @klyve weighing in...

Reply:
@icholy removed in 991edf1